balancer-js/src/modules/pools/pool-types/concerns/linear/exit.concern.ts

```diff
@@ -49,17 +50,7 @@ export class LinearPoolExit implements ExitConcern {
     wrappedNativeAsset,
     singleTokenOut,
   }: ExitExactBPTInParameters): ExitExactBPTInAttributes => {
-    // TODO implementation
-    console.log(
-      exiter,
-      pool,
-      bptIn,
-      slippage,
-      shouldUnwrapNativeAsset,
-      wrappedNativeAsset,
-      singleTokenOut
-    );
-    throw new Error('To be implemented');
+    throw new Error('Exit type not supported');
```

Comment:
Looking at the monorepo, these methods are not supported by this type of pool, so I updated the error messages to be more clear that no work is pending here.